frontend/src/pages/organization/SecretManagerOverviewPage/components/AllProjectView.tsx

```diff
@@ -0,0 +1,286 @@
+import { useState } from "react";
+import { useForm } from "react-hook-form";
+import {
```

Comment:
looks like we're missing the display of the proper validation error here?

<img width="1502" alt="image" src="https://github.com/user-attachments/assets/da771ed3-fbcf-4441-9fc6-8985cae3e870" />

Reply:
the subtitle implies that the note is optional but looks like we're requiring it
<img width="1303" alt="image" src="https://github.com/user-attachments/assets/646aac72-7229-440f-8371-737d32c74481" />

Reply:
It's optional, missed in backend